addon/components/file-upload-dialog.js

```diff
@@ -0,0 +1,29 @@
+import Component from '@ember/component';
+import layout from '../templates/components/file-upload-dialog';
+import { inject as service } from '@ember/service';
+import { A } from '@ember/array';
+
+export default Component.extend({
+  layout,
+  classNames: ['cs-component-file-upload-dialog'],
+  attributeBindings: ['dataTestName:data-test-cs-component-file-upload-dialog'],
+  dataTestName: true,
+  files: A([]),
+  fileQueue: service(),
+  actions: {
+    async uploadFile(file) {
+      const now = new Date();
+      file.uploadedDate = `${now.getMonth()}.${now.getDay()}.${now.getFullYear()}`;
```

Comment:
* months are 0-indexed, so you'll want to add 1
* `getDay()` returns the day of the week, you want `getDate()` for day of the month